Docs/clarinet js sdk updates (#1185)
hirosystems/clarinet: docs/feature-guides/clarinet-js-sdk.md

# Clarinet SDK

The Clarinet SDK is a JavaScript library to spawn and interact with a simulated Clarinet environment, also known as simnet.

A Simnet is a simulated network that simulates the Stacks blockchain and runs the Clarity VM, without the need for actual Stacks and Bitcoin nodes (unlike Devnet, Testnet and Mainnet).

Here is a non-exhaustive list of some of the use-cass:

- Call public and read-only functions from smart contracts
- Get clarity maps or data-var values

> - Get contract ABI

Reviewer: The references to "ABI" may not be a common convention to some developers. Perhaps worth expanding on the acronym somewhere in the page.

Reviewer 2: Made two suggestions too fully remove ABI. I don't it's 100% technically correct anyway

> - Get contract ABI

Reviewer 2: ```suggestion
- Get contract interfaces (available functions and data)
```

> Returns the interfaces of the project contracts.

Reviewer: Some parts of this language can be revisited (in a different PR, perhaps) for tonality. cc: @max-crawford

Reviewer 2: @sabbyanandan 
Resolving, we need to merge this PR to fix docs build

It returns a Map of Contracts, the keys are the contract addresses.

> The interfaces contain information such as the ABI, the available functions, data-vars and maps, NFT, and the FTs defined in the contract.

Reviewer 2: ```suggestion
The interfaces contain information such as the ABI, the available functions, data-vars and maps, NFTs, and the FTs defined in the contract.
```

> The interfaces contains informations such as the ABI, the available functions, data-vars and maps, the NFT and FT defined in the contract .

Reviewer: ```suggestion
The interfaces contain information such as the ABI, the available functions, data-vars and maps, NFT, and the FTs defined in the contract.
```

> The interfaces contain information such as the ABI, the available functions, data-vars and maps, NFTs, and the FTs defined in the contract.

Reviewer 2: ```suggestion
The interfaces contain information such as the available functions, data-vars and maps, NFTs, and the FTs defined in the contract.
```
